### PR TITLE
Superglue

### PR DESCRIPTION
**Pull request by yalov**
fixes https://github.com/zer0Kerbal/Kaboom/issues/5

### CODE REVIEW

**Comment by yalov** [comment]
`Welding.GetOffset` based on 
https://github.com/UmbraSpaceIndustries/Konstruction/blob/18b39deb9eb0ffe6c2c66da97629d0f631c9d1da/Source/Konstruction/Konstruction/Welding/ModuleWeldablePort.cs#L271
doesn't work properly

File: Source/Welding.cs [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+                if (!silent)
+                    ScreenMessages.PostScreenMessage("This part is the root part!  Cancelling");
+                return null;
+            }
+
+            return wData;
+        }
+
+
+        private Vector3 GetOffset(WeldingData wData)
```
Comment (yalov) [inline Source/Welding.cs:79]: Welding.GetOffset based on
https://github.com/UmbraSpaceIndustries/Konstruction/blob/18b39deb9eb0ffe6c2c66da97629d0f631c9d1da/Source/Konstruction/Konstruction/Welding/ModuleWeldablePort.cs#L271
doesn't work properly

File: source/Kaboom.cs [hunk 2]
```diff
@@ -28,23 +28,21 @@ public class ModuleKaboom : PartModule
         [KSPField(isPersistant = true)]
         public bool isGlued = false;
 
-        [KSPField(guiName = "Superglue", guiActive = true, guiActiveEditor = true, groupName = "Kaboom")]
+        [KSPField(isPersistant = true, guiName = "Superglue", guiActive = true, guiActiveEditor = true, groupName = "Kaboom")]
```
Comment (zer0Kerbal) [inline source/Kaboom.cs:31]: @yalov "Superglue it" love it - keeps up the kampy/cheeky/not taking itself too seriously attitude of KaboOom!

**Comment by yalov** [comment]
works in the simple stack
https://streamable.com/r8xj6m

**Comment by yalov** [comment]
`guiActiveUncommand` make KSPEvent active for incontrollable vessels, like debris tank

File: Source/Welding.cs (hunk 1 above)
Comment (zer0Kerbal) [reply Source/Welding.cs:77]: have to start somewhere though. 👍

**Comment by yalov** [comment]
https://streamable.com/1kx1km
check and demand parts, attached to nodes

**Comment by yalov** [comment]
I put the red color of the cover in the settings, but I don't really like it, and prefer to leave only normal text cover and remove this setting, what do you think?

**Comment by zer0Kerbal** [comment]
> I put the red color of the cover in the settings, but I don't really like it, and prefer to leave only normal text cover and remove this setting, what do you think?

I was going to put an option (might already be there) for `Colored PAW` in Game Settings. That way you can have it colored or not. Easy enough to do, done it elsewhere. 🎙️

**Review by zer0Kerbal — APPROVED** [review]
looking - well, better than if you superglued it.

**Review by zer0Kerbal — APPROVED** [review]
I will localize the settings later.

**Review by zer0Kerbal — APPROVED** [review]

**Review by zer0Kerbal — APPROVED** [review]
💯

**Review by zer0Kerbal — APPROVED** [review]
please tell me when you feel comfortable enough with the changes that I can compile a semi-public beta test.

**Comment by yalov** [comment]
It is ready for test

**Comment by zer0Kerbal** [comment]
Version 1.3.99.0 - BETA!-1 - New Feature
@zer0Kerbal zer0Kerbal released this 21 seconds ago

Mod Version
KSP version KSP-AVC License GPLv2 
Code Validate AVC .version files

Version 1.3.99.0 - BETA!
DO A CLEAN INSTALL: DELETE EXISTING THEN RE-INSTALL

BETA SOFTWARE - CAVEAT EMPTOR
⬇️ Download the .dll on Curseforge here.
New Feature: Kyanoacrylate 🦸 🩹 🧷 4️⃣ 🚀
usual spring cleaning and automation

[source] update folder structure
@yalov 
update to modern back-end automation

resolves issue #6

Update
recompile for KSP 1.12.2
update to .NET 4.7.2
update to C# 9.0
renamed ModuleKaboom.cs to Kaboom.cs
See Full Changelog for full details of changes
See Known Issues for known issues

red box below is a link to forum post on how to get support
How to get support

**Pull request merged**